src/util/fetchUtils.js

```diff
@@ -78,23 +79,31 @@ export const getFetchArgs = (apiUrl, options, queries, meta) => {
 		}
 
 		// send other metadata in searchParams
-		fetchUrl.searchParams.append('metadata', rison.encode_object(metadata));
+		if (metadata) {
+			fetchUrl.searchParams.append('metadata', rison.encode_object(metadata));
+		}
+	}
 
+	if (!isFormData) {
+		// need to manually specify content-type for any non-multipart request
+		headers['content-type'] = isPost && 'application/x-www-form-urlencoded' ||
```

Comment:
the content-type is set here in the browser for requests to the app server, but is set in `apiUtils` for requests to the API server.

It might be possible to avoid setting the `content-type` in `apiUtils`, but it can be a pretty fragile setting so I'd like to figure that out in a followup